src/utils/index.ts

```diff
@@ -45,10 +45,9 @@ export function isTouch() {
   );
 }
 
-export const dataAttr = (condition: boolean | undefined) =>
+export const dataAttr = (condition?: boolean) =>
   (condition ? "" : undefined) as Booleanish;
 
-export const ariaAttr = (condition: boolean | undefined) =>
-  condition ? true : undefined;
+export const ariaAttr = (condition?: boolean) => (condition ? true : undefined);
```

Comment:
Why this change 🤔 , any reasons?

I think line 51 can remain the same.

Reply:
Is this intentional that you used type `boolean | undefined`??

because the difference between `boolean | undefined` vs `condtion?: boolean` is first one expects to be undefined or boolean value and the argument cannot be empty.

but `condition?: boolean` argument can be empty and can pass undefined.

![image](https://user-images.githubusercontent.com/35374649/96563334-741d0d00-12df-11eb-80df-1d350b56aeda.png)
![image](https://user-images.githubusercontent.com/35374649/96563397-85feb000-12df-11eb-846f-b6934596e052.png)

Reply:
Based on the usage, both should be `boolean | undefined`.

We need the `condition` in both utils to be passed

Reply:
I made a mistake, just now gave it a thought !!

Reply:
boolean | undefined is totally fine if you intentionally want to restrict the arguments but both would not effect any logic behaviour since if we don't provide the argument it will be undefined anyways.

Reply:
Yeah, it would not affect the behaviour, we are just making it compulsory to pass a condition.